tests/unittests/test_firewatch_objects_rule.py

```diff
@@ -0,0 +1,141 @@
+        assert failure_type == "pod_failure"
+
+        # Test value "test_failure"
+        test_rule_dict = {"failure_type": "test_failure"}
+        failure_type = Rule._get_failure_type(self, test_rule_dict)
+        assert failure_type == "test_failure"
+
+    def test_get_classification(self) -> None:
+        test_rule_dict = {"classification": "Test classification"}
+        classification = Rule._get_classification(self, test_rule_dict)
+        assert classification == "Test classification"
+
+    def test_get_jira_project(self) -> None:
+        test_rule_dict = {"jira_project": "TEST"}
+        jira_project = Rule._get_jira_project(self, test_rule_dict)
+        assert jira_project == "TEST"
+
+    def test_get_jira_epic(self) -> None:
+        # Test when jira_epic is defined
+        test_rule_dict = {"jira_epic": "TEST-1234"}
+        jira_epic = Rule._get_jira_epic(self, test_rule_dict)
+        assert jira_epic == "TEST-1234"
+
+        # Test when jira_epic is not defined
+        test_rule_dict = {"step": "test"}
+        jira_epic = Rule._get_jira_epic(self, test_rule_dict)
+        assert jira_epic is None
+
+    def test_get_jira_component(self) -> None:
+        # Test when single component is defined
+        test_rule_dict = {"jira_component": "test component"}
+        jira_component = Rule._get_jira_component(self, test_rule_dict)
+        assert isinstance(jira_component, list)
+        assert "test component" in jira_component
+
+        # Test when multiple components are defined
+        test_rule_dict = {"jira_component": ["test component 1", "test component 2"]}
+        jira_component = Rule._get_jira_component(self, test_rule_dict)
+        assert isinstance(jira_component, list)
+        assert ("test component 1" in jira_component) and (
+            "test component 2" in jira_component
+        )
+
+        # Test when no components are defined
+        test_rule_dict = {"step": "test"}
+        jira_component = Rule._get_jira_component(self, test_rule_dict)
+        assert jira_component is None
+
+    def test_get_jira_affects_version(self) -> None:
+        # Test when jira_affects_version is defined
+        test_rule_dict = {"jira_affects_version": "test version"}
+        jira_affects_version = Rule._get_jira_affects_version(self, test_rule_dict)
+        assert jira_affects_version == "test version"
+
+        # Test when jira_affects_version is not defined
+        test_rule_dict = {"step": "test"}
+        jira_affects_version = Rule._get_jira_affects_version(self, test_rule_dict)
+        assert jira_affects_version is None
+
+    def test_get_ignore(self) -> None:
+        # Test when defined as "true"
+        test_rule_dict = {"ignore": "true"}
+        ignore = Rule._get_ignore(self, test_rule_dict)
+        assert ignore
+
+        # Test when defined as True
+        test_rule_dict = {"ignore": True}
+        ignore = Rule._get_ignore(self, test_rule_dict)
+        assert ignore
```

Comment:
Do you mean to separate into two different tests like` test_get_ignore_true` and `test_get_ignore_false`?

Reply:
What I meant was reducing code, by checking both `true, True` with or condition if its possible.  Since you are matching the dict and not strings, its fine to use what you have